OrbitLinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -41,6 +41,24 @@ void UprobesUnwindingVisitor::visit(SamplePerfEvent* event) {
   listener_->OnCallstack(returned_callstack);
 }
 
+void UprobesUnwindingVisitor::visit(SampleCallChainPerfEvent* event) {
+  CHECK(listener_ != nullptr);
+
+  if (current_maps_ == nullptr) {
+    return;
+  }
+
+  // TODO(kuebler): handle the mapping/patching for uprobes
+
+  // TODO(kuebler): we only have the IP here, however, that should be sufficient
```

Comment:
Why is it a TODO if it should be sufficient?

Reply:
I left it being a todo, to have later a double check about the post processing. However, as there is now another todo for this, I'll remove this here.